storage/rocksdb/ha_rocksdb.cc

```diff
@@ -6615,6 +6615,10 @@ static bool rocksdb_show_status(handlerton *const hton, THD *const thd,
 static bool rocksdb_lock_hton_log(
     handlerton *const MY_ATTRIBUTE((__unused__))) {
   assert(rdb != nullptr);
+  // FIXME(laurynas): workaround MyRocks bug of hanging if pfs.log_status is
```

Comment:
Reminder that https://github.com/facebook/rocksdb/pull/11020 address this issue. We should remove this once we update the submodule.

Reply:
Right, this is as temporary as it gets already. The purpose of this change was to make the 4th patch work in isolation work. The 6th patch is already reverting this.